ci-operator/templates/openshift/installer/cluster-launch-installer-libvirt-e2e.yaml

```diff
@@ -1,6 +1,7 @@
 kind: Template
```

Comment:
@sallyom I thought we agreed that you'll split the patches? This is 3 logical changes so it should be 3 commits. (Also typo in the log: `4)` -> `3)`.

Reply:
ok, done

Reply:
done